lib/main.js

```diff
@@ -87,7 +87,7 @@ export const endpoints = {
  * @param {Map} [options.implementations=allImplementations] - A Map of
  *   implementations.
  * @param {Function<boolean>} options.filter - A function to
- * filter the map's entries that returns true or false.
+ *  filter the map's entries that returns true or false.
```

Comment:
See also, linen 28, above
```suggestion
 *  filter the map's entries, that returns true or false.
```